src/compliance-rules.js

```diff
@@ -218,6 +223,37 @@ function addPopover(activityId, bpmnVisualization) {
     trigger: "click",
   });
 
+  // Add mouseover and mouseout event listeners to the table rows
+  tippyInstance.popper.addEventListener("mouseover", (event) => {
+    if (event.target.nodeName === "TD") {
+      const selectedRow = event.target.parentElement;

```

Comment:
**suggestion:** https://github.com/process-analytics/bonita-day-demo-2023/pull/18/files registers the event listener directly in the elements of the HTML of the popover. This is easy to register them directly on the rows (TR elements).

Reply:
I will take this into account

Reply:
fixed in 6e0c3006abeffc892f1c31bd58aa63c1bb32e551